app/views/head-lice.nunjucks

```diff
@@ -0,0 +1,133 @@
+{% import "_macros/images.nunjucks" as images %}
+
+{% extends '_layouts/content-simple.nunjucks' %}
+
+{% block local_header %}
+  <div class="local-header--intro">
+    <p>Head lice, also called nits, are very common in young children.
+      They don’t have anything to do with dirty hair.</p>
+  </div>
+{% endblock %}
+
+{% block main_content_before %}
+  <div class="page-section page-section__collapse-bottom">
+    <div class="reading-width">
+      <h2>Check if you have them</h2>
+    </div>
+
+    <div>
```

Comment:
@Demotive This also feels a bit messy but because we're using the lobotomized owl to only add a `margin-top` if the element has a preceding sibling we need to add an additional element to wrap the images as we are overriding the top margin to handle their spacing more accurately. 

This feel okay to you or do you have a neater alternative?